README.md

```diff
@@ -149,7 +149,7 @@ In our case the Github application keep up to date `dwylbot` with any events hap
 - Github App name: The name of the app; must be unique, so can't be "dwylbot" as that's taken!
 - Descriptions: A short description of the app; "My dwylbot app"
 - Homepage URL: The website of the app: "https://dwyl.com/"
-- User authorization callback URL: Redirect the user to this url for authentication each time the application is installed; "http://localhost:4000/auth/github/callback"
+- User authorization callback URL: Redirect url after user authentication e.g."http://localhost:4000/auth/github/callback". This is not needed for dwylbot so this field can be left empty.
```

Comment:
could we completely remove this bullet point instead of just updating it to say we don't need it?
or simplify it further to: 
+ User authorization callback URL Not needed by dwylbot (_leave it blank_)